### PR TITLE
Updates based on the UX feedback

### PR DESCRIPTION
**Pull request by zhx828**
This fixes https://github.com/oncokb/oncokb-pipeline/issues/601

### CODE REVIEW

File: src/main/webapp/app/shared/nav/StickyMiniNavBar.tsx [hunk 1]
```diff
@@ -161,11 +161,11 @@ export default function StickyMiniNavBar({
...
 
   return (
-    <Container
+    <div
       className={classNames(
-        'container',
         styles.container,
-        isSticky ? styles.containerSticky : ''
+        isSticky ? styles.containerSticky : '',
+        'container'
```
Comment (calvinlu3) [inline src/main/webapp/app/shared/nav/StickyMiniNavBar.tsx:168]: Whats the difference between the class and the component?

File: src/main/webapp/app/shared/icons/ShowHideToggleIcon.tsx
Comment (calvinlu3) [inline src/main/webapp/app/shared/icons/ShowHideToggleIcon.tsx:1]: The icon inside the circle doesn't seem to be centered. Seems to be slightly leaning left. Do you see that too?

File: src/main/webapp/app/shared/nav/StickyMiniNavBar.tsx (hunk 1 above)
Comment (zhx828) [reply src/main/webapp/app/shared/nav/StickyMiniNavBar.tsx:168]: Good question. Initially I removed the container, then added back the class. Let me try to revert to the component.

File: src/main/webapp/app/shared/icons/ShowHideToggleIcon.tsx
Comment (zhx828) [reply src/main/webapp/app/shared/icons/ShowHideToggleIcon.tsx:1]: Oh yeah, it might relate to some changes after introducing the icon. I will look inot.

Comment (zhx828) [reply src/main/webapp/app/shared/icons/ShowHideToggleIcon.tsx:1]: Not sure why I can't reproduce locally.

Comment (calvinlu3) [reply src/main/webapp/app/shared/icons/ShowHideToggleIcon.tsx:1]: I was looking at the screenshots, but locally it is fine for me too.

File: src/main/webapp/app/shared/nav/StickyMiniNavBar.tsx [hunk 2]
```diff
@@ -161,11 +161,11 @@ export default function StickyMiniNavBar({
   )[0]?.id;
 
   return (
-    <div
+    <Container
       className={classNames(
+        'container',
```
Comment (calvinlu3) [inline src/main/webapp/app/shared/nav/StickyMiniNavBar.tsx]: Can remove class now

**Review by calvinlu3 — APPROVED** [review]
LGTM. Screenshots still failing.

**Pull request merged**